src/dialog/_dialog.scss

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "../variables";
+@import "../mixins";
+
+.mdl-dialog {
+    display: block;
+    @include dialog-width();
+    margin: auto;
+    background: white;
+    position: fixed;
+    top: 5%;
+    right: 0;
+    left: 0;
+    z-index: 11;
+    @include shadow-24dp();
+
+    @media (min-height: 475px) {
+      top: 35%;
+    }
+
+    &:not([open]) {
+        display:none;
+    }
+
+    &::backdrop{
+        background-color: $dialog-backdrop-color;
+    }
+
+    &-backdrop {
+        background-color: $dialog-backdrop-color;
+        position:fixed;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        z-index: 10;
+        height: 100%;
+        width: 100%;
+    }
+
+    &__title {
+        padding-left: 24px;
+        padding-right: 24px;
+        padding-top: 24px;
+    }
+    &__actions {
+        padding: 8px 8px 8px 24px;
+        display: flex;
+        flex-direction: row-reverse;
+        flex-wrap: wrap;
+        * {
+            margin-right: 8px;
+            height: 36px;
+            &:last-child {
+                margin-right: 0;
+            }
+        }
+        &--full-width {
+          padding: 0 0 8px 0;
+          * {
```

Comment:
Interesting link. Thanks for that, @web2style. However, none of the reasons listed apply to us here. We are only supporting IE9 and above and MDL expects the whole page to be loaded anyways, before any JS is being executed which is relevant in the contexts of dialogs. 

I’ll leave it up to @Garbee to decide if he wants to switch to `:first-child`, but I don’t see a benefit.